docassemble/ALToolbox/al_income.py

```diff
@@ -545,8 +553,8 @@ class ALExpenseList(ALIncomeList):
     A list of expenses
 
     * each element has a:
+        * value
         * source
-        * owner
```

Comment:
Expenses don't set `.owner` in any questions, and it generally doesn't make sense for an expense to have an "owner". Might be the case on specific lines of credit or loans, but not general expenses like groceries.

Reply:
Yes, exactly. I'm saying ALIncomeList may have the same erroneous documentation.

Reply:
Ah, no, that's intentional. The full text there is "It can make use of these attributes and methods in its items", meaning owner is an attribute of ALIncome.